datafusion/expr/src/udf_docs.rs

```diff
@@ -149,21 +149,35 @@ impl DocumentationBuilder {
     ///
     /// This is similar to  [`Self::with_argument`] except that  a standard
     /// description is appended to the end: `"Can be a constant, column, or
```

Comment:
The example for the standard description I think can be removed as it's redundant with the examples below (and it's not quite correct)